addon/content/es-modules/ui.js

```diff
@@ -28,55 +28,38 @@ let RTK = null;
 let PropTypes = null;
 let useState = null;
 
-if (typeof nodeRequire === "function") {
-  /*global nodeRequire*/
-  //
-  // Node.js Environment
-  //
-  // If `nodeRequire` is defined, we are in a node environment
-  // (most likely for testing) and we should use the node require function.
-  // If we don't, we risk having multiple copies of React loaded at once.
-  React = nodeRequire("react");
-  ReactDOM = nodeRequire("react-dom");
-  Redux = nodeRequire("redux");
-  ReactRedux = nodeRequire("react-redux");
-  RTK = nodeRequire("@reduxjs/toolkit");
-  PropTypes = nodeRequire("prop-types");
-  useState = nodeRequire("react").useState;
-} else {
-  //
-  // Browser/Thunderbird Environment
-  //
-  // In a browser environment, we simulate AMD module loading so that we
-  // can pass in dependencies appropriately.
-  const initializedDeps = {};
+//
+// Browser/Thunderbird Environment
+//
+// In a browser environment, we simulate AMD module loading so that we
+// can pass in dependencies appropriately.
+const initializedDeps = {};
 
-  // These names must be in the same order as the AMD modules were imported
-  const amdModuleNames = [
-    "react",
-    "react-dom",
-    "redux",
-    "react-redux",
-    "redux-toolkit",
-    "prop-types",
-  ];
-  amdModuleNames.forEach((name, i) => {
-    const amdItem = amdModules[i];
-    if (!amdItem) {
-      throw new Error(
-        `An ${i}th AMD module was assumed to be loaded, but none was found`
-      );
-    }
-    initializedDeps[name] = callWithDeps(amdItem, initializedDeps);
-  });
+// These names must be in the same order as the AMD modules were imported
+const amdModuleNames = [
+  "react",
+  "react-dom",
+  "redux",
+  "react-redux",
+  "redux-toolkit",
+  "prop-types",
+];
+amdModuleNames.forEach((name, i) => {
+  const amdItem = amdModules[i];
+  if (!amdItem) {
+    throw new Error(
+      `An ${i}th AMD module was assumed to be loaded, but none was found`
+    );
+  }
+  initializedDeps[name] = callWithDeps(amdItem, initializedDeps);
+});
 
-  React = initializedDeps.react;
-  ReactDOM = initializedDeps["react-dom"];
-  Redux = initializedDeps.redux;
-  ReactRedux = initializedDeps["react-redux"];
-  RTK = initializedDeps["redux-toolkit"];
-  PropTypes = initializedDeps["prop-types"];
-  useState = initializedDeps.useState;
-}
+React = initializedDeps.react;
+ReactDOM = initializedDeps["react-dom"];
+Redux = initializedDeps.redux;
+ReactRedux = initializedDeps["react-redux"];
+RTK = initializedDeps["redux-toolkit"];
+PropTypes = initializedDeps["prop-types"];
+useState = initializedDeps.useState;
 
 export { React, ReactDOM, Redux, ReactRedux, RTK, PropTypes, useState };
```

Comment:
The initial idea of this file was to put all the UI exports/imports in one place so that external libraries could be imported from a single place. It looks like it's not needed anymore...? Can we just get rid of it?

Reply:
Most likely, still got a lot of cleaning to do :)